tsconfig.json

```diff
@@ -7,6 +7,7 @@
 		"noUnusedLocals": true,
 		"noImplicitReturns": true,
 		"noFallthroughCasesInSwitch": true,
+		"esModuleInterop": true,
```

Comment:
This can be safely deleted, everything else LGTM. :+1:

Reply:
Unfortunately it can't - https://github.com/nfriedly/express-rate-limit/pull/282#discussion_r779435279

Reply:
I think we can let this remain as now we don't need users to use the `esModuleInterop` flag, only the library needs to use it to make sure we can run the tests.